pallets/avn/src/tests/test_proxy_signed_add_ethereum_logs.rs

```diff
@@ -21,83 +23,31 @@ mod test_set_ethereum_contract {
             Context {
                 origin: RawOrigin::Root.into(),
                 new_contract_address: H160::from([15u8; 20]),
-                ethereum_contract: EthereumContracts::ValidatorsManager,
+                ethereum_contract: EthereumContracts::EthereumBridgeContract,
             }
         }
     }
 
     impl Context {
-        fn dispatch_set_ethereum_contract(&self) -> DispatchResult {
-            return EthereumEvents::set_ethereum_contract(
+        fn dispatch_set_ethereum_contract(&self, address: H160) -> DispatchResult {
+            return AVN::<TestRuntime>::set_bridge_contract(
                 self.origin.clone(),
-                self.ethereum_contract.clone(),
-                self.new_contract_address.clone(),
+                address.clone(),
             )
         }
     }
 
     mod successful_cases {
         use super::*;
         #[test]
-        fn update_validators_manager_contract() {
+        fn update_contract() {
             let mut ext = ExtBuilder::build_default().with_genesis_config().as_externality();
             ext.execute_with(|| {
                 let context = Context::default();
-                assert_ne!(
-                    context.new_contract_address,
-                    EthereumEvents::get_contract_address_for_non_nft_event(
-                        &ValidEvents::AddedValidator
-                    )
-                    .unwrap()
-                );
```

Comment:
I don't think this touches the ValidatorsManager though. The Validator event is a placeholder, in fact, and I made a comment to change the function altogether not to use it. But there are multiple issues here.

One is, as you say, that in this file it no longer makes sense to test this function in particular since we only kept the nft contracts. That was not clear when I wrote the comment.

Another issue is that this function exists somewhere else, but no longer needs an argument. 

The other though is that in the context of fixing this test but not removing, only this part was removed. Inasmuch as the test still exists, removing the pre-condition feels wrong, and I wrote the comment to teach that part, this is, why we have these pre-conditions in place. Michael is still starting, so I use these initial PRs as talking points to explain general practices and, for example, test-writing philosophy.